app/src/main/java/com/github/se/gomeet/InitFunctions.kt

```diff
@@ -124,17 +130,31 @@ fun initChatClient(applicationContext: Context): ChatClient {
 @Composable
 fun InitNavigation(nav: NavHostController, client: ChatClient, applicationContext: Context) {
   val navAction = NavigationActions(nav)
-  val userIdState = remember { mutableStateOf("") }
   val clientInitialisationState by client.clientState.initializationState.collectAsState()
   val authViewModel = AuthViewModel()
-  val eventViewModel = remember { mutableStateOf(EventViewModel(null)) }
   val userViewModel = UserViewModel()
+  val startScreen = Route.WELCOME // The screen that gets navigated to when the app starts
+  val postLoginScreen =
+      Route.EXPLORE // The screen that gets navigated to after logging in/signing up
```

Comment:
In your PR description, it is written that you changed the screen the user goes in after login to `Trends` but here it navigates to `Explore`, correct ?

Reply:
Well spotted, I forgot to update the PR's description (the navigation to Explore is intentional)